packages/@aws-cdk/aws-codebuild/lib/report-group.ts

```diff
@@ -46,6 +46,7 @@ abstract class ReportGroupBase extends cdk.Resource implements IReportGroup {
         'codebuild:CreateReport',
         'codebuild:UpdateReport',
         'codebuild:BatchPutTestCases',
+        'codebuild:BatchPutCodeCoverages',
```

Comment:
This one I actually am less sure of... can you please revert this change?

Reply:
@skinny85 .Took care of it. Had some issue with my local builds had to rely on AWS Codebuild. Please check. 
I also think the report-group.ts should be updated to support codecoverage soon to maintain parity.

Reply:
Perhaps, but that's a separate issue (predicated on the fact that currently [`ReportGroup` hard-codes the `Type` property to `'TEST'`](https://github.com/aws/aws-cdk/blob/cf268218163a6d25cc6bbe7375b9332f87062631/packages/%40aws-cdk/aws-codebuild/lib/report-group.ts#L128-L128)).

Reply:
Yeah, I am happy to contribute to that as well. Let me know if you would like to have a separate issue reported and tracked to closure?

Reply:
Yes please 🙂